src/app/content/highlights/components/SummaryPopup/HighlightsHelpInfo.tsx

```diff
@@ -32,6 +32,7 @@ const slideInAnimation = css`
 const Wrapper = styled.div`
   position: fixed;
   bottom: 0;
+  left: 0;
```

Comment:
why is this added here?

Reply:
`HighlightsHelpInfo` was previously outside of the `<Modal>` and now it is inside